packages/models/tests/ratings.test.ts

```diff
@@ -79,7 +79,9 @@ describe("Ratings", () => {
         values: [1, 2, 4, 5],
       });
 
-      const avg = await ratings.findAvgRatings([t1.id, t2.id, t3.id]);
+      const avg = await ratings.findAvgRatings({
```

Comment:
unrelated changes, Try to rebase on latest master

Reply:
![image](https://github.com/user-attachments/assets/c2e11a44-b003-46f2-9252-e960f2cbad99)

@mostafakamar2308 
I want to ask about that point 
findAvgRatings function take an object as a parameter containing users as an array of numbers so how can we give the function this array directly without the object and label of users

Reply:
@mostafakamar2308 
I have tried to rebase on the latest master but the error is still present

Reply:
If you rebased now, you should not face this issue.